pages/_app.tsx

```diff
@@ -201,85 +201,87 @@ const PlanetWeb = ({ Component, pageProps }: any) => {
     return <BrowserNotSupported />;
   } else {
     return (
-      <ErrorHandlingProvider>
-        <div>
-          <div
-            style={
-              showVideo &&
-              (config.tenantName === 'planet' || config.tenantName === 'ttc')
-                ? {}
-                : { display: 'none' }
-            }
-          >
-            {config.tenantName === 'planet' || config.tenantName === 'ttc' ? (
-              <VideoContainer setshowVideo={setshowVideo} />
-            ) : (
-              <></>
-            )}
-          </div>
+      <div>
+        <div
+          style={
+            showVideo &&
+            (config.tenantName === 'planet' || config.tenantName === 'ttc')
+              ? {}
+              : { display: 'none' }
+          }
+        >
+          {config.tenantName === 'planet' || config.tenantName === 'ttc' ? (
+            <VideoContainer setshowVideo={setshowVideo} />
+          ) : (
+            <></>
+          )}
+        </div>
 
-          <div
-            style={
-              showVideo &&
-              (config.tenantName === 'planet' || config.tenantName === 'ttc')
-                ? { display: 'none' }
-                : {}
-            }
+        <div
+          style={
+            showVideo &&
+            (config.tenantName === 'planet' || config.tenantName === 'ttc')
+              ? { display: 'none' }
+              : {}
+          }
+        >
+          <Auth0Provider
+            domain={process.env.AUTH0_CUSTOM_DOMAIN}
+            clientId={process.env.AUTH0_CLIENT_ID}
+            redirectUri={process.env.NEXTAUTH_URL}
+            audience={'urn:plant-for-the-planet'}
+            cacheLocation={'localstorage'}
+            onRedirectCallback={onRedirectCallback}
+            useRefreshTokens={true}
           >
-            <Auth0Provider
-              domain={process.env.AUTH0_CUSTOM_DOMAIN}
-              clientId={process.env.AUTH0_CLIENT_ID}
-              redirectUri={process.env.NEXTAUTH_URL}
-              audience={'urn:plant-for-the-planet'}
-              cacheLocation={'localstorage'}
-              onRedirectCallback={onRedirectCallback}
-              useRefreshTokens={true}
-            >
-              <ThemeProvider>
-                <MuiThemeProvider theme={materialTheme}>
-                  <CssBaseline />
-                  <QueryParamsProvider>
-                    <UserPropsProvider>
+            <ThemeProvider>
+              <MuiThemeProvider theme={materialTheme}>
+                <CssBaseline />
+                <QueryParamsProvider>
+                  <UserPropsProvider>
+                    <ErrorHandlingProvider>
```

Comment:
`ErrorHandlingProvider` should be at the top level in order to handle errors arising from anywhere in the app. 

@Shreyaschorge please correct me if I'm wrong here.

Reply:
I got an error , so that's why I changed the order of errorhandlingprovider in component tree

Reply:
@sunilsabatp What was the final call taken here?

Reply:
the errorhandling provider placed to it's earlier position